### PR TITLE
DISABLE_SSL option added (#133).

### PR DESCRIPTION
**Pull request by litnimax**
Added. Tested:
```
root@mars:/home/dokku-postgres# DISABLE_SSL=1 dokku postgres:create lolo22
       Waiting for container to be ready
       Creating container database
       Not securing connection to database
=====> Postgres container created: lolo22
=====> Container Information
       Config dir:          /var/lib/dokku/services/postgres/lolo22/config
       Data dir:            /var/lib/dokku/services/postgres/lolo22/data
       Dsn:                 postgres://postgres:87e9f3a43ec5cc185ff1cfbf7f452c28@dokku-postgres-lolo22:5432/lolo22
       Exposed ports:       -
       Id:                  3f3bb713b67d93fe584b79692dbd6f5c8d15156c8d5432bd9c094cc5cc2aef3f
       Internal ip:         172.17.0.8
       Links:               -
       Service root:        /var/lib/dokku/services/postgres/lolo22
       Status:              running
       Version:             postgres:9.6.4
root@mars:/home/dokku-postgres#
```

And

```
root@mars:/home/dokku-postgres# dokku postgres:create lolo23
       Waiting for container to be ready
       Creating container database
       Securing connection to database
=====> Postgres container created: lolo23
=====> Container Information
       Config dir:          /var/lib/dokku/services/postgres/lolo23/config
       Data dir:            /var/lib/dokku/services/postgres/lolo23/data
       Dsn:                 postgres://postgres:ec5c2daa7e5df13bd04c8fdbea3819ac@dokku-postgres-lolo23:5432/lolo23
       Exposed ports:       -
       Id:                  389caac26ab815beb52fda011119eed62612bca6fcfc42236357815753a47c80
       Internal ip:         172.17.0.9
       Links:               -
       Service root:        /var/lib/dokku/services/postgres/lolo23
       Status:              running
       Version:             postgres:9.6.4
root@mars:/home/dokku-postgres#
```

### CODE REVIEW

File: functions [hunk 1]
```diff
@@ -70,9 +70,14 @@ service_create_container() {
   DATABASE_NAME="$(get_database_name "$SERVICE")"
   docker exec "$SERVICE_NAME" su - postgres -c "createdb -E utf8 $DATABASE_NAME" 2> /dev/null || echo 'Already exists'
 
-  dokku_log_verbose_quiet "Securing connection to database"
-  service_stop "$SERVICE" > /dev/null
-  docker run --rm -i -v "$SERVICE_ROOT/data:/var/lib/postgresql/data" "$PLUGIN_IMAGE:$PLUGIN_IMAGE_VERSION" bash -s < "$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)/scripts/enable_ssl.sh" &> /dev/null
+
+  service_stop "$SERVICE" > /dev/null;
+  if [ -z "$DISABLE_SSL" ]; then
```
Comment (josegonzalez) [inline functions]: This needs a double-bracket.

File: functions [hunk 2]
```diff
@@ -70,9 +70,14 @@ service_create_container() {
   DATABASE_NAME="$(get_database_name "$SERVICE")"
   docker exec "$SERVICE_NAME" su - postgres -c "createdb -E utf8 $DATABASE_NAME" 2> /dev/null || echo 'Already exists'
 
-  dokku_log_verbose_quiet "Securing connection to database"
-  service_stop "$SERVICE" > /dev/null
```
Comment (josegonzalez) [inline functions]: Probably can move this  into the if statement.

**Comment by litnimax** [comment]
Why the test if failed?

File: functions [hunk 3]
```diff
@@ -70,9 +70,14 @@ service_create_container() {
...
-  docker run --rm -i -v "$SERVICE_ROOT/data:/var/lib/postgresql/data" "$PLUGIN_IMAGE:$PLUGIN_IMAGE_VERSION" bash -s < "$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)/scripts/enable_ssl.sh" &> /dev/null
+
+  service_stop "$SERVICE" > /dev/null;
+  if [[ -z "$DISABLE_SSL" ]]; then
```
Comment (josegonzalez) [inline functions]: This doesn't persist at the moment, so if the container is recreated, we'll re-enable ssl.

**Comment by josegonzalez** [comment]
Closing as there wasn't any follow-up. Feel free to update the PR and I'll take a look again.

**Pull request closed without merging**